docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -113,3 +113,4 @@
 - mssql-jtds-1.x
 - mssql-jdbc
 - apache-cxf-3.x
+- seata-1.4.x
```

Comment:
I think you don't need this. This list is a native plugin list. You should update `supported-list.md`.